### PR TITLE
Quick, Map, and List pools should stop calculating allocated bytes and high water marks.

### PR DESCRIPTION
**Pull request by mcfadden8**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
This should be handled by the AllocationTracker/Inspector...?

**Comment by mcfadden8** [comment]
> This should be handled by the AllocationTracker/Inspector...?

Agreed.  As dicussed, I have renamed the ticket writtten for this as well as changed the title of this PR.

**Comment by mcfadden8** [comment]
@davidbeckingsale, please review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 ### Changed
 
 ### Removed
```
Comment (davidbeckingsale) [inline CHANGELOG.md:15]: Add a blank line here.

Comment (mcfadden8) [reply CHANGELOG.md:15]: Done.

**Pull request merged**